docs/reference/style/pack.rst

```diff
@@ -465,3 +458,104 @@ specifying the allocated width and allocated height.
+
+Pack aims to be a functional subset of CSS. Any Pack layout can be converted
+into an equivalent CSS layout. After applying this conversion, the CSS layout
+should be considered a "reference implementation". Any disagreement between the
+rendering of a converted Pack layout in a browser, and the layout produced by
+the Toga implementation of Pack should be considered to be either a bug in Toga,
+or a bug in the mapping.
+
+The mapping that can be used to establish the reference implementation is:
+
+* The reference HTML layout document is rendered in `standards mode
+  <https://developer.mozilla.org/en-US/docs/Web/API/Document/compatMode>`__,
+  with a default CSS stylesheet:
+
+  .. code-block:: html
+
+      <!DOCTYPE html>
+      <html>
+         <head>
+           <meta charset="UTF-8" />
+            <title>Pack layout testbed</title>
+            <style>
+               html, body {
+                  height: 100%;
+                  white-space: nowrap;
+               }
+               body {
+                  overflow: hidden;
+                  display: flex;
+                  margin: 0;
+               }
+               div {
+                  display: inline-flex;
+                  margin: 0;
+               }
+            </style>
+         </head>
+         <body></body>
+      </html>
+
+* The root element of the Pack layout can be mapped to the ``<body>`` element of
+  the HTML reference document. The rendering area of the browser window becomes
+  the view area that Pack will fill.
+
+* All other elements in the DOM tree are mapped to ``<div>`` elements.
+
+* The following Pack declarations can be mapped to equivalent CSS declarations:
+
+   ============================= ===================================================
+   Pack property                 CSS property
+   ============================= ===================================================
+   ``alignment: top``            ``align-items: start`` if ``direction == row``;
+                                 otherwise ignored.
+   ``alignment: bottom``         ``align-items: end`` if ``direction == row``;
+                                 otherwise ignored.
+   ``alignment: left``           ``align-items: start`` if ``direction == column``;
+                                 otherwise ignored.
+   ``alignment: right``          ``align-items: end`` if ``direction == column``;
+                                 otherwise ignored.
+   ``alignment: center``         ``align-items: center``
+   ``direction: <str>``          ``flex-direction: <str>``
+   ``display: pack``             ``display: flex`` on the root element of a layout;
+                                 ``display: inline-flex`` on all other elements.
+   ``flex: <int>``               If ``direction = row`` and ``width`` is set,
+                                 or ``direction = column`` and ``height`` is set,
+                                 ignore. Otherwise, ``flex: <int> 0 0%``.
```

Comment:
I think it's confusing to use a % sign here, because it would be uncommon to use a percentage in this context. The examples in the standard [here](https://www.w3.org/TR/css-flexbox-1/#valdef-flex-flex-basis) and [here](https://www.w3.org/TR/css-flexbox-1/#flex-initial) simply say "0".

This also applies to the other % signs below, and in the `__css__` method in pack.py.

Reply:
Weird - I thought I saw something on MDN that rendered the default flex as `0 0 0%`... but now I can't find it. Updated here, and in `__css__` to use `0`.